clips/dry/docker-dry

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Docker Client Plugin Manager (CLIP)
+# Explore contents of the image layers with the Dive tool
+#
+# (c) 2019 Łukasz Lach
```

Comment:
:)

Reply:
i didn't know if i should changed that or not :)

Reply:
Yup, this one is yours